### ISSUE TITLE
unmasking value incorrect

### ISSUE THREAD

**Issue by sticcino** [plentz/jquery-maskmoney#248]
THIS:
"<script type=\"text/javascript\">$(\"#x_B_Offer_Price\").maskMoney({thousands:',', decimal:'.', precision: 0, allowZero: true, allowNegative:false, prefix: ' $'});</script>";

example: $900,000 input

when unmasked:
var num = $('#x_B_Offer_Price').maskMoney('unmasked')[0]; alert('type: '+ typeof(num) + ', value: ' + num)

results in: 900 NOT 900000

**Comment by RNAPolymerase** [comment]
This is an important issue. Perhaps the most important. I am running into this as well. The problem occurs when there is no decimal point included in the source number. This occurs regardless of whether or not a decimal character has been set up in initialization.

Please see my codesandbox for a working example of the issue: https://codesandbox.io/s/rlq0r8jwwn

**Comment by arthur5005** [comment]
Yeah this is some _wonky_ behaviour. Completely unexpected. *If the precision is zero*, the unmasked output should an *integer*, always.
